app/android/src/uk/co/lutraconsulting/InputActivity.java

```diff
@@ -115,6 +122,41 @@ public void hideSplashScreen()
     keepSplashScreenVisible = false;
   }
 
+public void showPDF(String filePath) {
```

Comment:
I would rename the method as it is now _file-type-agnostic_ :)
```suggestion
public void openFile(String filePath) {
```